CompetitionTemplate/src/competition/subsystems/drive/commands/TankDriveWithJoysticksCommand.java

```diff
@@ -5,19 +5,27 @@
 import competition.operator_interface.OperatorInterface;
 import competition.subsystems.drive.DriveSubsystem;
 import xbot.common.command.BaseCommand;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.XPropertyManager;
 
 
 public class TankDriveWithJoysticksCommand extends BaseCommand {
 
     final DriveSubsystem driveSubsystem;
     final OperatorInterface oi;
+    final BooleanProperty enableSquaredTankInputs;
     
     @Inject
-    public TankDriveWithJoysticksCommand(OperatorInterface oi, DriveSubsystem driveSubsystem)
+    public TankDriveWithJoysticksCommand(
+            OperatorInterface oi, 
+            DriveSubsystem driveSubsystem,
+            XPropertyManager propMan)
     {
         this.oi = oi;
         this.driveSubsystem = driveSubsystem;
         this.requires(this.driveSubsystem);
+        
+        enableSquaredTankInputs = propMan.createPersistentProperty("EnableSquaredTankInputs", false);
```

Comment:
Would maybe add in the name that this is for the drive with joysticks (vs if we used the gamepad this wouldn't affect it?)